release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -795,6 +795,25 @@ You can now upgrade a Metering Operator to 4.5 from 4.2 through 4.4. Previously,
 
 With this update, support for respecting a cluster-wide proxy configuration is available. Additionally, the upstream repository moved from the operator-framework organization to kube-reporting.
 
+[id="ocp-4-5-virtualization"]
+=== OpenShift Virtualization
+
+[id="ocp-4-5-virtualization-ga"]
+==== OpenShift Virtualization is now generally available
+
+Red Hat OpenShift Virtualization is now a fully supported and generally
```

Comment:
@codyhoag Last-minute change request: Can you please replace the first line:
Red Hat {VirtProductName} is now a fully supported and generally available feature of {product-title} 4.5. 
with
Red Hat {VirtProductName} is supported to run on {product-title} 4.5.

Reply:
@aburdenthehand I have removed the text that came immediately after as that also talks about GA.

Reply:
Thanks for resolving this and getting it merged!